tests/specs/InferFunctionAssignment.txt

```diff
@@ -0,0 +1,59 @@
+# mod.ts
+export function foo(n = 0) {}
```

Comment:
Maybe add a test here for `n: string = 0` just to see what it does. It should be `string`, right?